RevenueCatUI/CustomerCenter/Views/Virtual Currencies/VirtualCurrencyBalanceListRow.swift

```diff
@@ -59,7 +64,7 @@ struct VirtualCurrencyBalanceListRow: View {
 
     var body: some View {
         CompatibilityLabeledContent {
-            Text(rowData.virtualCurrencyCode)
+            Text("\(rowData.virtualCurrencyName) (\(rowData.virtualCurrencyCode))")
```

Comment:
What happens if this is too long?

Reply:
Good question. 

How about I add in:
```swift
.lineLimit(2)
.multilineTextAlignment(.leading)
```

so it can grow some, but not too much.

The only other place in Customer Center that we're setting a line limit is in a title in `FeedbackSurveyView`, which sets the line limit to 2. It uses `.multilineTextAlignment(.center)`, which is appropriate for a title, but I don't think it's super fitting to center text in a list like this.

Reply:
Added in 2795fde2e8239a525ebd3bd1aa18e616c318b315 !